servers/services/src/main/java/org/projectnessie/services/rest/TreeResource.java

```diff
@@ -186,6 +212,33 @@ public LogResponse getCommitLog(String ref, CommitLogParams params)
     }
   }
 
+  private Stream<ImmutableCommitMeta> filterWithCelExpression(
+      Stream<ImmutableCommitMeta> items, CommitLogParams params) {
+    return items.filter(
+        commit -> {
+          try {
+            Script script =
```

Comment:
Move this out of the Stream.
No need to parse+check the same expression for every CommitMeta.
That also allows to properly propagate parse+check errors back to the caller (`ScriptCreateException`).

Reply:
yeah it was rather a quick hack to get something running. Will move it out in the next comit